.github/workflows/markdown-lint.yml

```diff
@@ -7,6 +7,8 @@ on:
     paths:
       - .nvmrc
       - .markdownlint-cli2.jsonc
+      - .markdownlint.jsonc
+      - "**/.markdownlint.jsonc"
```

Comment:
I think we could just use:

```suggestion
      - "**/.markdownlint.jsonc"
```

documentation: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths

![image](https://github.com/mdn/translated-content/assets/15844309/936a0fda-9334-433e-b436-d0b30574b66c)